.github/workflows/demos.yml

```diff
@@ -23,7 +23,7 @@ jobs:
       - name: Install Melos
         run: flutter pub global activate melos
       - name: Install dependencies
-        run: melos bootstrap
+        run: melos prepare
```

Comment:
I don't think this command exists ? 
<img width="164" alt="image" src="https://github.com/powersync-ja/powersync.dart/assets/46312751/f2a7a428-31f8-4899-bc0b-3b73f846b135">

Reply:
It exists after being defined [here](https://github.com/powersync-ja/powersync.dart/blob/25a75f0cb5372381628f2990c9cbde2c62163758/melos.yaml#L12)